.github/workflows/platformDeploy.yml

```diff
@@ -31,40 +31,12 @@ jobs:
     if: ${{ fromJSON(needs.validateActor.outputs.IS_DEPLOYER) }}
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: Expensify/App/.github/actions/composite/setupNode@main
 
-      - uses: actions/setup-ruby@v1
+      - uses: ruby/setup-ruby@08245253a76fa4d1e459b7809579c62bd9eb718a
```

Comment:
Note the use of https://github.com/ruby/setup-ruby instead of deprecated https://github.com/actions/setup-ruby.